pkgs/by-name/ly/lyssa/package.nix

```
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  taglib,
+  cglm,
+  libclipboard,
+  libxcb,
+  makeWrapper,
+  python3Packages,
+  ffmpeg,
+  yt-dlp,
+  miniaudio,
+  jq,
+  glfw,
+  exiftool,
+  stb,
+  libGL
+}:
+stdenv.mkDerivation {
+  name = "lyssa";
+  version = "unstable-2024-06-01";
+  src = fetchFromGitHub {
+    owner = "cococry";
+    repo = "lyssa";
+    rev = "a7ebb786b005d9f716dfa99869e29660ca14c749";
+    hash = "sha256-jJx7V/ru6quvZhqkVK3OGguH81882fQXNdpLVp+g5sk=";
+  };
+
+  buildInputs = [ exiftool stb glfw miniaudio yt-dlp jq ffmpeg cglm libclipboard libxcb libGL ];
+  nativeBuildInputs = [ pkg-config taglib makeWrapper ];
+
+  postFixup = ''
+    wrapProgram $out/bin/lyssa \
+      --prefix PATH ${python3Packages.glad}/bin/glad
```

I'm pretty sure you want to put the directory in PATH not the binary?
```suggestion
      --prefix PATH ${python3Packages.glad}/bin
```

---

Yeah, I realize there's a much better way of doing that 
```nix
wrapProgram $out/bin/lyssa \
  --prefix PATH : ${
   lib.makeBinPath [ python3Packages.glad ]
}
```